### PR TITLE
adds missing embed_map

### PR DESCRIPTION
**Pull request by javisantana**
please @Kartones, this thing :(

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2062/))

File: app/controllers/api/json/oembed_controller.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def show
     end
 
     # build the url using full schema because any visuaization should work with any user
-    url = CartoDB.user_url(user, organization)  + public_visualizations_show_path(id: uuid)
+    url = CartoDB.user_url(user, organization)  + public_visualizations_show_path(id: uuid) + "/embed_map"
```
Comment (Kartones) [inline app/controllers/api/json/oembed_controller.rb]: Uhmmm, something's wrong here:

From the routes.rb file:
```
# Oembed
    get '(/u/:user_domain)/api/v1/oembed' => 'oembed#show', as: :api_v1_oembed
```
So the correct line would be:
```
url = CartoDB.user_url(user, organization)  + api_v1_oembed_path
```

But then... where's the visualization id placed?

I mean, a visualization-show url + /embed_map won't work with current routes.rb definitions

Comment (Kartones) [inline app/controllers/api/json/oembed_controller.rb]: No handcrafted url paths in rails please. Replace by this:
```
url = CartoDB.user_url(user, organization) + public_visualizations_embed_map_path(id:uuid)
```

**Comment by Kartones** [comment]
+1

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2063/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2064/))

**Pull request merged**